### PR TITLE
edit collaborations page

### PR DESCRIPTION
**Pull request by jukent**
Adds mini gallery of Cookbooks and larger sections for each collaboration. Text needs to be fleshed out and possibly some more visually appealing elements.

### CODE REVIEW

**Comment by philipc2** [comment]
Should we consider including UXarray's collaboration with the DOE SEATS Project and other efforts? Any thoughts @erogluorhan ?

**Comment by jukent** [comment]
@philipc2 it's set up so that editing `data/collaborations.yml` can add any number of sections.

**Comment by erogluorhan** [comment]
> Should we consider including UXarray's collaboration with the DOE SEATS Project and other efforts? Any thoughts @erogluorhan ?

Yes, definitely!

File: data/collaborations.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - title: "Advanced Visualization Cookbook"
+        url: "https://projectpythia.org/advanced-viz-cookbook/README.html"
+        image: "https://projectpythia.org/advanced-viz-cookbook/_images/2b484515ab7d60fb802616951a1c621d06a9200a93088fd268686eed11e1f982.png"
+
+      - title: Unstructured Grid Visualization Cookbook
+        url: https://projectpythia.org/unstructured-grid-viz-cookbook/README.html
+        image: https://raw.githubusercontent.com/ProjectPythia/unstructured-grid-viz-cookbook/main/thumbnail.png
+
+  - title: "CUPiD"
+    description: "CUPiD stands for the CESM Unified Postprocessing and Diagnostics. This project handles a lot of data standartdization and computation for CESM."
```
Comment (philipc2) [inline data/collaborations.yml]:
```suggestion
    description: "CESM Unified Postprocessing and Diagnostics (CUPiD) is a Python framework for generating diagnostics from CESM that enables and integrates timeseries file generation, data standardization, diagnostics, and metrics from all CESM components."
```

For consistency with the blurb above on ESDS

> "Earth System Data Science (ESDS) is a community building effort at..."

Also modified the description slightly to align with the description online

File: data/collaborations.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+enable : true
+title  : GeoCAT Cookbooks
+collaborations:
+
+  - title: "ESDS"
+    description: "Earth System Data Science (ESDS) is a community building effort at NSF NCAR / UCAR to bring geoscientists and geoscientific software engineers together to share new tools, ideas, and techniques for the evolving best practices with big data Python."
```
Comment (philipc2) [inline data/collaborations.yml]:
```suggestion
    description: "Earth System Data Science (ESDS) is a community building effort at NSF NCAR / UCAR that brings geoscientists and geoscientific software engineers together to share new tools, ideas, and techniques for the evolving best practices with big data Python."
```

File: content/collaborations/_index.md [hunk 3]
```diff
@@ -1,10 +1,7 @@
 ---
-title : "Collaborations"
-date  : 2022-04-27T00:00:00-06:00
-image : images/banners/cloudSimulation.png
+title: "Collaborations"
+date: 2022-04-27T00:00:00-06:00
+image: images/banners/cloudSimulation.png
+description: "GeoCAT team members are involved in many other community software engineering efforts. Here are our most frequent collaborations:"
```
Comment (philipc2) [inline content/collaborations/_index.md]:
```suggestion
description: "The GeoCAT team is involved in many community software engineering efforts. Here are our most frequent collaborations:"
```

File: content/news/ams2025.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Register for GeoCAT’s short course at AMS 2025!"
```
Comment (philipc2) [inline content/news/ams2025.md]: _**r/unexpectedfactorial**_

This is a very slight nit-pick, but consider dropping the exclamation point from the year?

**Review by philipc2 — CHANGES_REQUESTED** [review]
Nice work! A few small suggestions.

**Comment by philipc2** [comment]
> @philipc2 it's set up so that editing `data/collaborations.yml` can add any number of sections.

Sounds good! I'll open up a PR with UXarray collaborations once this one is merged.

File: content/news/ams2025.md (hunk 4 above)
Comment (jukent) [reply content/news/ams2025.md]: Don't want it to look like a factorial

**Comment by jukent** [comment]
Thanks for the edits @philipc2 I'm going to merge this now so that you can work on the UXarray addition.

**Pull request merged**